pkgs/development/python-modules/astroquery/default.nix

```diff
@@ -6,25 +6,27 @@
 , keyring
 , beautifulsoup4
 , html5lib
+, numpy
 , pytest
 , pytest-astropy
+, pytest-astropy-header
 , astropy-helpers
 }:
 
 buildPythonPackage rec {
   pname = "astroquery";
-  version = "0.3.10";
+  version = "0.4";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1ce57a8792c7d5d74206d797d379de6da35d56be433ea5840c41a49f202e2fab";
+    sha256 = "1ddwnj9vpvxkrfb45c4pwv5f5za9kn2q0040dpw2ymj2bwlpl61h";
   };
 
-  propagatedBuildInputs = [ astropy requests keyring beautifulsoup4 html5lib ];
+  propagatedBuildInputs = [ astropy requests keyring beautifulsoup4 html5lib numpy ];
```

Comment:
I don't think that `numpy` is needed here. `astroquery` depends on `astropy` and `astropy` itself depends on `numpy`.

Reply:
Done, thanks.

Reply:
if it's listed in install-requires, you'll want to keep it. astropy is free to drop a dependency in the future.

Reply:
There's no install requires in setup.py. But numpy is imported in various places and their own CI seems to always install it (https://github.com/astropy/astroquery/blob/7fa89014d2743028f4263473b5b0343e0e6400a4/.travis.yml). Please advice.